query10.sql

```diff
@@ -1,7 +1,26 @@
+-- Johnathan Clementi --
+-- MUSA 509 Geospatial Cloud Computing & Vizualization --
+-- HW #1 2021-09-19 --
+-- Query #10 --
 /*
   Using the station status dataset, find the distance in meters of all stations
   from Meyerson Hall.
 */
 
 -- Enter your SQL query here
-select ...
+select 
+  name,
+  trunc(st_distance(
+    st_transform(st_setsrid(st_makepoint(-75.19265, 39.95224),4326),3857),
+    st_transform(the_geom,3857) 
```

Comment:
Remember that SRID 3857 is going to be pretty distorted at latitudes far from the equator. Even though the order below is correct (because everything around this latitude is distorted _the same amount_, so the distances are correct _relative to each other_), the actual _absolute_ distances are over-inflated.